src/app/modules/users/store/user.effects.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { TestBed } from '@angular/core/testing';
+import { provideMockActions } from '@ngrx/effects/testing';
+import { Action } from '@ngrx/store';
+import { Observable, of, throwError } from 'rxjs';
+import { UsersService } from '../services/users.service';
+import { UserActionTypes } from './user.actions';
+import { UserEffects } from './user.effects';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ToastrModule, ToastrService } from 'ngx-toastr';
+
+describe('UserEffects', () => {
+  let actions$: Observable<Action>;
+  let effects: UserEffects;
+  let service: UsersService;
+  let toastrService;
+  const user = { id: 'id', name: 'name', email: 'email' };
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [UserEffects, provideMockActions(() => actions$)],
+      imports: [HttpClientTestingModule, ToastrModule.forRoot()],
+      declarations: [],
+    });
+    effects = TestBed.inject(UserEffects);
+    service = TestBed.inject(UsersService);
+    toastrService = TestBed.inject(ToastrService);
+  });
+
+  it('should be created', async () => {
+    expect(effects).toBeTruthy();
+  });
+
+  it('should return a list of users when the action LOAD_USERS call succeeds', async () => {
```

Comment:
The test name does not reflect what you're testing here. It should be:

```
 action type is LOAD_USER_SUCCESS when service is executed sucessfully 
```

Reply:
the description has been changed thanks